Add initial intro for Bevy 0.14 release notes (#1198)
bevyengine/bevy-website: content/news/draft-bevy-0.14/index.md

public_draft = 1188
+++

> Thanks to **TODO** contributors, **TODO** pull requests, community reviewers, and our [**generous sponsors**](/community/donate), we're happy to announce the **Bevy 0.14** release on [crates.io](https://crates.io/crates/bevy)!

Reviewer: I assume these TODO are intentionally left in?

Author: Yep, we can't get the final numbers until commits are frozen.

For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out our [Quick Start Guide](/learn/quick-start) to try it today. It's free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.

> And to see what the engine has to offer hands-on, check out the entries in the [latest Bevy Jam](https://itch.io/jam/bevy-jam-4/entries), including the winner [That's a LOT of beeeeees](https://andrewb330.itch.io/thats-a-lot-of-beeeeees).

Reviewer 2: Are these meant to be two separate paragraphs, or one? I believe the Markdown spec requires two newlines between paragraphs.

Author: ```suggestion
For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out our [Quick Start Guide](/learn/quick-start) to try it today. It's free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.
```

That second line shouldn't exist IMO.

To update an existing Bevy App or Plugin to **Bevy 0.14**, check out our [0.13 to 0.14 Migration Guide](/learn/migration-guides/0-13-to-0-14/).

Since our last release a few months ago we've added a _ton_ of new features, bug fixes, and quality of life tweaks, but here are some of the highlights:

- **Shiny 3D rendering features:** depth of field, god rays, meshlets, motion blur and more! Bevy's 3D rendering is quickly closing the feature gap.
- **Better colors:** type-safe colors make it clear which color space you're operating in, and offer an awesome array of useful methods.

> - **Animation blending:** our new procedural animation graph unblocks animation blending, and sets us up for graphical, asset-driven animation tools.

Author: ```suggestion
- **Animation blending:** our new low-level animation graph unblocks animation blending, and sets the stage for first- and third-party graphical, asset-driven animation tools.
```

- **Observers and hooks:** automatically respond to component addition and removal as we lay the foundations for fast, reliable relations between entities.

> - **Computed states and substates:** modelling complex app state is a breeze with these type-safe extensions to our `States` abstraction.

Reviewer 2: ```suggestion
- **Computed states and substates:** modeling complex app state is a breeze with these type-safe extensions to our `States` abstraction.
```

I believe American spelling was used in the past, though I don't have a large preference on it. Feel free to dismiss this nit if you prefer the double-l.

> - **Rounded corners:** rounding off one of `bevy_ui`'s roughest corners.

Reviewer: It’d be good to add a sentence of what more specifically can be rounded, for example “all mode bundles can now have a corner radius specified”.
